### ISSUE TITLE
Exomedia demo play the same song when back press and select the another song

### ISSUE THREAD

**Issue by sujitkjha** [brianwernick/ExoMedia#145]
I am new in android and I have downloaded the exoplayer demo app. I have 5 songs in the list view, 4 songs from server and one from sdcard.

when I start playing first time by clicking the list item, player start playing the song, when I press back button and play another song the same previous song get played.

Please please Please help me

**Comment by TonyTangAndroid** [comment]
It seems that the demo in ExoMedia branch 3.0 also has such a bug.

**Comment by brianwernick** [comment]
This is due to a missing line in the demo.  I have updated the `master` branch to include the fix and will update `3.0` shortly.

In the `AudioPlayerActivity` you need to update the `startPlayback()` method to be 

```java
    private void startPlayback(boolean forceStart) {
        //If we are changing audio files, or we haven't played before then start the playback
        if (forceStart || playlistManager.getCurrentIndex() != selectedIndex) {
            playlistManager.setCurrentIndex(selectedIndex);
            playlistManager.play(0, false);
        }
    }
```
